src/sx-tailwind/src/generate.js

```diff
@@ -11,25 +11,62 @@ import generateSX from './sxGenerator';
 const css = fs.readFileSync(path.join(__dirname, 'tailwind.css'));
 const styles = generateSX(css);
 
-const code = SignedSource.signFile(`
+const codeStyles = SignedSource.signFile(`
 /**
  * ${SignedSource.getSigningToken()}
  * @flow
  *
- * Tailwind CSS file converted into SX stylesheets
+ * Tailwind CSS file converted into stylesheets for SX
  *
  * @see https://tailwindcss.com/
  * @see https://github.com/adeira/sx
  */
 
+export const tailwindStyles = Object.freeze(${JSON.stringify(styles)});
+`);
+
+const codeTypes = SignedSource.signFile(`
+/**
+ * ${SignedSource.getSigningToken()}
+ * @flow
+ *
+ * Union type of all Tailwind CSS classes
+ *
+ * @see https://tailwindcss.com/
+ */
+
+export type TailwindClassNames = "${Object.keys(styles).join('"|"')}";
```

Comment:
Please, try to check if you can improve the Flow performance, as we discussed privately.

Reply:
Done

Reply:
What about the Flow cache on CI? Would you be able to set it up? Or better to create a new issue?

Reply:
New issue I'd say

Reply:
https://github.com/adeira/universe/issues/1078

Reply:
And PR here: https://github.com/adeira/universe/pull/1079

Reply:
I closed the issue and PR since it's not possible to do it in Github CI for now. Feel free to proceed with this PR.